spec/controllers/idv/hybrid_mobile/socure/document_capture_controller_spec.rb

```diff
@@ -175,6 +176,79 @@
         expect(response).to be_not_found
       end
     end
+
+    context 'when socure error encountered' do
+      let(:fake_socure_endpoint) { 'https://fake-socure.com/' }
+      let(:failed_response_body) do
+        { 'status' => 'Error',
+          'referenceId' => '1cff6d33-1cc0-4205-b740-c9a9e6b8bd66',
+          'data' => {},
+          'msg' => 'No active account is associated with this request' }
+      end
+      let(:response_body_401) do
+        {
+          status: 'Error',
+          referenceId: '7ff0cdc5-395e-45d1-8467-0ff1b41c11dc',
+          msg: 'string',
+        }
+      end
+      let(:no_doc_found_response_body) do
+        {
+          referenceId: '0dc21b0d-04df-4dd5-8533-ec9ecdafe0f4',
+          msg: {
+            status: 400,
+            msg: 'No Documents found',
+          },
+        }
+      end
+      before do
+        allow(IdentityConfig.store).to receive(:socure_document_request_endpoint).
+          and_return(fake_socure_endpoint)
+      end
+      it 'connection timeout still responds to user' do
+        stub_request(:post, fake_socure_endpoint).to_raise(Faraday::ConnectionFailed)
+        get(:show)
+
+        expect(response).not_to be_nil
```

Comment:
```suggestion
        expect(response.body).to eq('')
        expect(response).to be_ok
```
in our specs we generally use:
        expect(response).to be_ok
or 
        expect(response.status).to eq(200)